lib/ldap_fluff/netiq.rb

```diff
@@ -0,0 +1,44 @@
+class LdapFluff::NetIQ < LdapFluff::Generic
+
+  def bind?(uid = nil, password = nil, opts = {})
+    unless uid.include?(',') || opts[:search] == false
+      service_bind
+      user = @member_service.find_user(uid)
+      uid = user.first.dn if user && user.first
```

Comment:
Is the condition necessary? If I'm reading it right then https://github.com/theforeman/ldap_fluff/pull/71/files#diff-150a012639e8faaebe318dc258205828794c64cc2241d64e595326d501bd8959R13 either returns an array with at least one argument or raises an exception

Reply:
Obsolete for this PR now, but still a good point for a new PR :wink: